deb/DEBIAN/changelog

```diff
@@ -0,0 +1,5 @@
+sdl_hmi (6.1.0) stable; urgency=medium
+
+* Testing.
```

Comment:
@LuxoftAKutsan I believe you can put here something like "Initial commit/version" as this is not a testing already

Reply:
Fixed